code/game/objects/structures/windows/window.dm

```diff
@@ -90,6 +92,24 @@
 	if (flags_1 & ON_BORDER_1)
 		AddElement(/datum/element/connect_loc, loc_connections)
 
+/obj/structure/window/mouse_drop_receive(atom/dropping, mob/user, params)
+	. = ..()
```

Comment:
If the parent proc isn't doing anything then it isn't nessassary

Reply:
Well, it handles buckling, even if windows aren't something you can normally buckle to.